app/src/main/java/com/example/themealsapp/domain/model/Meal.kt

```diff
@@ -0,0 +1,51 @@
+package com.example.themealsapp.domain.model
+
+
+data class Meal(
+    val idMeal: String,
+    val strMeal: String,
+    val strArea: String,
+    val strCategory: String,
+    val strInstructions: String,
+    val strYoutube: String,
+    val strIngredient1: String,
+    val strIngredient2: String,
+    val strIngredient3: String,
+    val strIngredient4: String,
+    val strIngredient5: String,
+    val strIngredient6: String,
+    val strIngredient7: String,
+    val strIngredient8: String,
+    val strIngredient9: String,
+    val strIngredient10: String,
+    val strIngredient11: String,
+    val strIngredient12: String,
+    val strIngredient13: String,
+    val strIngredient14: String,
+    val strIngredient15: String,
+    val strIngredient16: String?,
```

Comment:
if it is domain try not to have null values.  If it is needed try to have default values on nullables

Reply:
I will try to change this in a next version, because I have already tried to work on it, and it did not work out.